images/plbase/python-requirements.txt

```diff
@@ -21,13 +21,13 @@ pygraphviz==1.13
 pyquaternion==0.9.9
 pytest-cov==5.0.0
 pytest-repeat==0.9.3
-pytest==8.2.1
+pytest==8.2.2
 recommonmark==0.7.1
 regex==2024.5.15
-ruff==0.4.7
+ruff==0.5.0
```

Comment:
We were impacted by a change in this new version: https://astral.sh/blog/ruff-v0.5.0#changes-to-e999-and-reporting-of-syntax-errors. See the changes to `pyproject.toml` that account for this.